cudamapper/src/minimizer.cu

```diff
@@ -906,11 +911,11 @@ Minimizer::GeneratedSketchElements Minimizer::generate_sketch_elements(DefaultDe
     device_buffer<char> window_minimizers_direction_d(total_windows, allocator, cuda_stream);
     GW_LOG_INFO("Allocating {} bytes for window_minimizers_position_in_read_d", total_windows * sizeof(position_in_read_t));
     device_buffer<position_in_read_t> window_minimizers_position_in_read_d(total_windows, allocator, cuda_stream);
-    GW_LOG_INFO("Allocating {} bytes for read_id_to_minimizers_written_d", number_of_reads_to_add * sizeof(std::uint32_t));
-    device_buffer<std::uint32_t> read_id_to_minimizers_written_d(number_of_reads_to_add, allocator, cuda_stream);
+    GW_LOG_INFO("Allocating {} bytes for read_id_to_minimizers_written_d", number_of_reads_to_add * sizeof(std::int64_t));
+    device_buffer<std::int64_t> read_id_to_minimizers_written_d(number_of_reads_to_add, allocator, cuda_stream);
```

Comment:
Wouldn't `int32_t` be large enough?

Reply:
I was thinking about this as well. This array is also used as an output of `inclusive_sum()` for it should be able to keep the total number of minimizers in an index. I think 31 bits should still be enough for this (otherwise matcher would probably go OOM on every system in foreseeable future), but I'm not sure about mapping to long reference.
As this array is expected to have a few thousand elements I would keep it 64-bit just in case, if you agree

Reply:
Makes sense. Thanks for the additional explanation.